Script/AtomicEditor/hostExtensions/languageExtensions/TypescriptLanguageExtension.ts

```diff
@@ -456,7 +459,9 @@ export default class TypescriptLanguageExtension implements Editor.HostExtension
             if (result.type == "success") {
                 message += `<color #00ff00>${result.text}</color>`;
             } else {
-                message += `<color #e3e02b>${result.text} at line ${result.row} col ${result.column}</color>`;
+                message += `<color #e3e02b>${result.text} at line ${result.row + 1} col ${result.column}</color> <widget TBSkinImage: skin: MagnifierBitmap, text:"..." id: link${linkId}>`;
+                links["link" + linkId] = result;
```

Comment:
Nice use of MagnifierBitmap 🔍, looking forward to building this out and trying it :)